contracts/voting/cwd-voting-cw721-staked/src/error.rs

```diff
@@ -3,33 +3,30 @@ use thiserror::Error;
 
 #[derive(Error, Debug, PartialEq)]
 pub enum ContractError {
-    #[error("{0}")]
+    #[error(transparent)]
     Std(#[from] StdError),
 
     #[error("Nothing to claim")]
     NothingToClaim {},
 
-    #[error("Invalid token")]
+    #[error("Invalid token. Got ({received}), expected ({expected})")]
     InvalidToken { received: Addr, expected: Addr },
 
     #[error("Unauthorized")]
     Unauthorized {},
```

Comment:
can we make this err a bit more specific?

Reply:
great point. updated to specify that the issue at hand is that the sender is not the owner.